### PR TITLE
added fresher's news item

### PR DESCRIPTION
**Pull request by FromAnkyra**

### CODE REVIEW

File: news/2020-09-01-freshers-fair.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+One of our biggest concessions to the current times has been, and will
+continue to be for quite some time, in-person events. Our society revolves around the knowledge and enthusiasm of our members, and our members’ willingness to involve others, both in our academic events and our social ones, and this is harder online than it is in person. In the end, however, the safety of the members of our society comes before our need for members, or our want for physical events.
+
+Given our outlook, an outlook shared by other societies we’re sure, 
+we were more than a bit shocked to see YUSU choosing to go ahead with
+plans for an in-person Freshers’ Fair. We were even more shocked to 
+learn about the plans from York Vision, rather than from YUSU directly.
+It seems YUSU cares more about publicity than the societies it 
+represents being well informed.
+On face value the plans for an in-person Freshers’ Fair make
```
Comment (sersorrel) [inline news/2020-09-01-freshers-fair.md:21]: missing blank line between these paragraphs

**Comment by LukeMoll** [comment]
Most recent news items as shown on the front page

Might be nice to add a short lede before the first paragraph.

File: news/2020-09-01-freshers-fair.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+Going into this academic year, during the current unprecedented times,
+we were, as a student society, aware that there would be sacrifices.
+Our new student takeup and retention were likely to be lower, our 
+events more restricted, funding more difficult to come by, and many 
+other considerations. We planned for these as best we could, and where
+we couldn’t make things work, we reformed them or cut them.
+
+One of our biggest concessions to the current times has been, and will
+continue to be for quite some time, in-person events. Our society revolves around the knowledge and enthusiasm of our members, and our members’ willingness to involve others, both in our academic events and our social ones, and this is harder online than it is in person. In the end, however, the safety of the members of our society comes before our need for members, or our want for physical events.
```
Comment (JMAlego) [inline news/2020-09-01-freshers-fair.md:13]: Are we missing some hard-wrapping here to meet the 70 character guide?

**Review by JMAlego — CHANGES_REQUESTED** [review]
One small hard-wrapping issue and I agree with @LukeMoll that we should have a lede (which you are already working on, I'm aware).

**Comment by FromAnkyra** [comment]
@LukeMoll added

File: news/2020-09-01-freshers-fair.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
+---
+title: The 2020 Fresher's Fair
+---
+
+YUSU have released [a statement](https://www.yorkvision.co.uk/news/exclusive/01/09/2020) about how Freshers' Fair will go forth this year. We have opinions.
```
Comment (JMAlego) [inline news/2020-09-01-freshers-fair.md:5]: This line hasn't been hard-wrapped to 70 characters, but the link makes that hard.

**Review by JMAlego — APPROVED** [review]
I'm fine with this as is, or you can make the suggested change.

File: news/2020-09-01-freshers-fair.md (hunk 3 above)
Comment (sersorrel) [reply news/2020-09-01-freshers-fair.md:5]: [reference-style link noises] :^)

Comment (sersorrel) [reply news/2020-09-01-freshers-fair.md:5]: i.e.

```markdown
released [a statement][vision] about

[vision]: https://www.yorkvision.co.uk/news/exclusive/01/09/2020
```

**Pull request merged**